tasmota/tasmota.ino

```diff
@@ -337,7 +337,7 @@ void setup(void) {
     Settings->baudrate = APP_BAUDRATE / 300;
     Settings->serial_config = TS_SERIAL_8N1;
   }
-  SetSerialBaudrate(Settings->baudrate * 300);  // Reset serial interface if current baudrate is different from requested baudrate
+  SetSerialBegin();                            // Reset serial interface if current serial settings are different from requested serial settings
```

Comment:
Another try do ya. I tried to explain why this is wrong in a fix after your first pr. This breaks any which is in quick power cycle mode as it will corrupt flash. Don't do this.

Reply:
Whoops... didn't realize that.  Reversing changes on that hunk.